src/report/html.rs

```diff
@@ -68,6 +68,7 @@ impl ResultName for BrokenReason {
             BrokenReason::Unknown => "broken crate".into(),
             BrokenReason::CargoToml => "broken Cargo.toml".into(),
             BrokenReason::Yanked => "deps yanked".into(),
+            BrokenReason::PrivateGitRepository => "private repo".into(),
```

Comment:
While it's technically a private repository it will only happen on Crater for deleted repos. I'd just change this to "missing repo".